app/license.go

```diff
@@ -62,6 +62,10 @@ func (a *App) SaveLicense(licenseBytes []byte) (*model.License, *model.AppError)
 		return nil, model.NewAppError("addLicense", "api.license.add_license.unique_users.app_error", map[string]interface{}{"Users": *license.Features.Users, "Count": uniqueUserCount}, "", http.StatusBadRequest)
 	}
 
+	if license != nil && license.IsExpired() {
```

Comment:
Would be nice if there was a good way to unit test this without having to include an actual license in the tests